### PR TITLE
Change user status to an enum, better than magic numbers

### PR DESCRIPTION
**Pull request by paladine**
Branches from linebreaks, so merge that one first

### CODE REVIEW

File: MBBSEmu/HostProcess/Enums/UserStatus.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+namespace MBBSEmu.HostProcess.Enums
+{
+    /// <summary>
+    /// User Status values, mostly taken from BRKTHU.H
+    /// </summary>
+    public enum UserStatus
```
Comment (enusbaum) [inline MBBSEmu/HostProcess/Enums/UserStatus.cs:6]: Rename to `EnumUserStatus` to keep consistent Enum naming convention

File: MBBSEmu/HostProcess/Enums/UserStatus.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        UNUSED = 0, // default unused value
```
Comment (enusbaum) [inline MBBSEmu/HostProcess/Enums/UserStatus.cs:8]: You can use XML documentation on the enums to provide tool tips for their value while in an IDE

File: MBBSEmu/HostProcess/Enums/UserStatus.cs (hunk 1 above)
Comment (enusbaum) [inline MBBSEmu/HostProcess/Enums/UserStatus.cs:6]: I also don't _THINK_ it matters, but Enum's are typed as `int` by default, but we can specify ` : ushort` just to keep the type consistent.

**Review by enusbaum — CHANGES_REQUESTED** [review]

File: MBBSEmu/HostProcess/Enums/UserStatus.cs (hunk 1 above)
Comment (paladine) [reply MBBSEmu/HostProcess/Enums/UserStatus.cs:6]: I thought Polish notation was dead?

**Review by enusbaum — APPROVED** [review]

**Pull request merged**